src/lua/luaTask.c

```diff
@@ -241,6 +242,15 @@ static void lua_task(void *params)
 
                 /* If its a known unrecoverable, fail fast */
                 switch (rc) {
+		case LUA_NO_PERIODIC_FUNCTION:
```

Comment:
I'm thinking this will still allow a one time script to still execute?  just confirming.